utils.rb

```diff
@@ -0,0 +1,19 @@
+require "yaml"
+class Utils
+  attr_accessor :file_path
+
+  def initialize(file_path)
+    @path = file_path
+  end
+
+  def load_file
+    a = YAML.load_file @path
+  end
+
+  def write_file(destination)
+    file = File.open(destination, 'w')
+    file.write(load_file.to_yaml)    
```

Comment:
Please save data from app memory (@lib variable from `library.rb`) not from another file